### PR TITLE
Added batch update alerts documentaion

### PR DESCRIPTION
**Pull request by SujudAbu-Atta**

### CODE REVIEW

File: batch-update-alerts.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+title: Batch Update alert entities API
+description: Learn how to update Microsoft Defender ATP alerts in a batch by using this API. You can update the status, determination, classification, and assignedTo properties.
```
Comment (adirdidi) [inline batch-update-alerts.md:3]: "Microsoft Defender ATP" -> "Microsoft Defender for Endpoint".

File: batch-update-alerts.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+[!include[Microsoft Defender for Endpoint API URIs for US Government](../../includes/microsoft-defender-api-usgov.md)]
+
+[!include[Improve request performance](../../includes/improve-request-performance.md)]
+
+
+## API description
+Updates properties of a batch of existing [Alerts](alerts.md).
+<br>Submission of **comment** is available with or without updating properties.
+<br>Updatable properties are: ```status```, ```determination```, ```classification``` and ```assignedTo```.
```
Comment (illfated) [inline batch-update-alerts.md:37]: The document looks quite good. May I just recommend to use only a single pair of MarkDown back ticks around single words or strings on the same line? 3 pairs of back ticks is usually only needed for MD code block fences.
```suggestion
<br>Updatable properties are: `status`, `determination`, `classification` and `assignedTo`.
```

**Review by illfated — APPROVED** [review]
Looks pretty good to me.

**Comment by Dansimp** [comment]
@mjcaparas please review

**Comment by illfated** [comment]
To the team member who will be merging the content from this PR: please check if it is possible to edit the commit title to correct the typo (documentaion).

**Comment by garycentric** [comment]
@mjcaparas 

_batch-update-alerts.md_ is in the root of the repo, and the link to it in _alerts.md_ provides no path, so the location is assumed to be the same as that of _alerts.md_. The result is a build warning in sync PR https://github.com/MicrosoftDocs/windows-docs-pr/pull/4713. Also, _batch-update-alerts.md_ should be listed in a TOC file, but that change is not included in this PR.

**Pull request merged**